File: Programacion_OO/Presentacion/metodos.py
class Persona:
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad = edad

    def saludar(self):  # Método
        return f"¡Hola, mi nombre es {self.nombre} y tengo {self.edad} años!"

# Instanciación de objetos
persona1 = Persona("Juan", 30)
persona2 = Persona("Ana", 25)

# Llamada al método
print(persona1.saludar())  # Llamamos al método 'saludar' de persona1
print(persona2.saludar())  # Llamamos al método 'saludar' de persona2

File: Programacion_OO/Presentacion/Metodos.js
class Persona {
    constructor(nombre, edad) {
        this.nombre = nombre;
        this.edad = edad;
    }

    saludar() {  // Método
        return `¡Hola, mi nombre es ${this.nombre} y tengo ${this.edad} años!`;
    }
}

// Instanciación de objetos
const persona1 = new Persona("Juan", 30);
const persona2 = new Persona("Ana", 25);

// Llamada al método
console.log(persona1.saludar());  // Llamamos al método 'saludar' de persona1
console.log(persona2.saludar());  // Llamamos al método 'saludar' de persona2
